bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/graphics/Resource.java

```diff
@@ -123,6 +124,10 @@ public Resource() {
 void destroy() {
 }
 
+void destroyHandlesExcept(Set<Integer> zoomLevels) {
```

Comment:
Add some documentation here and mention the fact that the default implementation is empty and therefore sub-classes should override this method if necessary.

Reply:
Alright

Reply:
How about:

"Destroy all handles of the resource  which are <strong>not</strong> necessary for the given zoom levels. This method is supposed to be overridden by sub-classes that retain handles for different zoom levels."

For the body? 

The `@param` is ok.

Reply:
But wer dont know what zoom levels these resources have handles of. So, the only way possible is to tell them to keep these zoom levels and discard others if they have.

Reply:
alright